beekeeper-api/pom.xml

```diff
@@ -20,6 +20,45 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-actuator</artifactId>
     </dependency>
+    <dependency>
+      <groupId>net.kaczmarzyk</groupId>
+      <artifactId>specification-arg-resolver</artifactId>
+      <version>2.6.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.graphql-java</groupId>
+      <artifactId>graphql-spring-boot-starter</artifactId>
+      <version>5.0.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.expediagroup</groupId>
```

Comment:
Can you put this dependency after the spring dependency? We tend to do our project ones at the top